src/SDK/Resource/Detectors/Host.php

```diff
@@ -15,13 +15,84 @@
  */
 final class Host implements ResourceDetectorInterface
 {
+    private const PATH_ETC_MACHINEID = 'etc/machine-id';
+    private const PATH_VAR_LIB_DBUS_MACHINEID = 'var/lib/dbus/machine-id';
+    private const PATH_ETC_HOSTID = 'etc/hostid';
+    public function __construct(
```

Comment:
Apologies @matt-hensley - I messed the spacing up with my previous suggestion!

```suggestion
    private const PATH_ETC_HOSTID = 'etc/hostid';

    public function __construct(
```